contrib/epee/include/storages/portable_storage.h

```diff
@@ -59,7 +61,7 @@ namespace epee
       bool       get_value(const std::string& value_name, t_value& val, hsection hparent_section);
       bool       get_value(const std::string& value_name, storage_entry& val, hsection hparent_section);
       template<class t_value>
-      bool       set_value(const std::string& value_name, const t_value& target, hsection hparent_section);
+      bool       set_value(const std::string& value_name, t_value&& target, hsection hparent_section);
```

Comment:
This would be much cleaner as `t_value target` rather than `t_value&& target` so that the caller can decide whether to move or copy the argument.  Then you can keep all the `std::move()` changes for efficiency while dropping all the places where the caller has to use `int64_t(val)` and the like to make explicit copies at the call site.

Reply:
Oh never mind, you're taking this as universal reference.  I was misreading the diff.